pkg/test/framework/components/istio/ingress/interface.go

```diff
@@ -88,23 +88,16 @@ type Instance interface {
 	// when in an evnironment that doesn't support LoadBalancer).
 	DiscoveryAddress() net.TCPAddr
 
-	// Call makes a call through ingress.
-	Call(options CallOptions) (CallResponse, error)
-	CallOrFail(t test.Failer, options CallOptions) CallResponse
-	CallWithRetry(options CallOptions, retryOptions ...retry.Option) (CallResponse, error)
-	CallWithRetryOrFail(t test.Failer, options CallOptions, retryOptions ...retry.Option) CallResponse
-
 	// CallEcho makes a call through ingress using the echo call and response types. This is recommended
```

Comment:
nit: you can remove " and the other method will likely be removed in the future."